src/pairing.cpp

```diff
@@ -157,33 +157,11 @@ Pair *check_good_pair(RoughReservoir* upper, RoughReservoir* lower,
   }
   ExistingPit single_pit;
 
-  if (search_config.search_type == SearchType::BULK_PIT) {
-    for (uint i = 0; i < pit_details.size(); i++) {
-      if (upper->brownfield && pit_details[i].reservoir.identifier == upper->identifier) {       
-        single_pit = pit_details[i];
-        break;
-      }
-      else if (lower->brownfield && pit_details[i].reservoir.identifier == lower->identifier) {
-        single_pit = pit_details[i];
-        break;
-      }
-      
-      // Throw error if there are no pit details assigned to single_pit
-      if (i == pit_details.size() - 1) {
-        string pit_id = "MISSING ID";
-        if (upper->brownfield)
-          pit_id = upper->identifier;
-        else
-          pit_id = lower->identifier;
-        search_config.logger.debug("No pit details in existing_reservoirs_csv for reservoir with ID: " + pit_id);
-        exit(1);  
-      }    
-    }
-  } else if (search_config.search_type == SearchType::SINGLE_PIT) {
```

Comment:
Pairing just doesn't require the pit_details object now - all information for the pit is stored in the processing_files/reservoirs/* CSV files now, rather than a separate pit_details file + Shapefile. So pairing uses the information in the processing_files/reservoirs/* files to match reservoirs.

Will also change the reservoir file to include the volume-altitude pairs that the old algorithm had.